packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/identity.constants.js

```diff
@@ -1,6 +1,7 @@
 export const REGEX_VALIDATORS = {
   SIRET: /^\d{14}$/,
   APE: /^\d{4}[a-zA-Z]$/,
+  URLWEBSITE: /(?:^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w.-]+)+[\w\-._~:/?#[\]@!$&'()*+,;=.]+$)|(?:^$)/,
```

Comment:
😭 
could we add a comment to explain what this regex corresponds to ?

Reply:
URLWEBSITE limits the insertion of invalid URLs. While allowing not to put http or https, but if it is present it requires "://". Also allows to leave empty.